app/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+const baseURL = "https://nab-grinder-converter.herokuapp.com";
+export default class App extends Component {
+  state = {
+    data: [],
+  };
+
+  componentDidMount() {
```

Comment:
Can this be done inside the Converter JSX component

Reply:
It can, but its still a prototype so the focus is functionality

Reply:
I asked because of easier state management
But since this is just for testing the functionality then it's fine